### ISSUE TITLE
Some errors :/

### ISSUE THREAD

**Issue by jpargudo** [fld/graph-apcupsd#1]
Hi there,

I tried your scripts this week-end, I had to adapt them (see files..), because apcaccess doesn't report to me OUTPUTV and ITEMP.. So I replaced it with BCHARGE (battery charge in %) and TIMELEFT (time left in minutes), wich I find usefull too.

I'm on Debian SID, and, 1st I had to add #!/bin/bash at the beginning of cron-apcupsd.sh to have it work.

I placed this script in /etc/cron.d/ to have it system-wide.

The log is okay:
```
# tail  /var/log/apcupsd/debug.log 
239.0:19.0:100.0:14.7:27.6:0:0
240.0:18.0:100.0:14.7:27.6:0:0
241.0:19.0:100.0:14.7:27.6:0:0
240.0:21.0:100.0:14.1:27.6:0:0
242.0:18.0:100.0:15.3:27.6:0:0
240.0:18.0:100.0:15.3:27.6:0:0
240.0:22.0:100.0:13.0:27.6:0:0
241.0:19.0:100.0:14.7:27.6:0:0
242.0:19.0:100.0:14.7:27.6:0:0
239.0:18.0:100.0:15.3:27.6:0:0
```

The rrdtool database was initiated with:
```
`rrdtool create /etc/apcupsd/apcupsd.rrd --step '60' \
'DS:LINEV:GAUGE:120:0:300'  \
'DS:LOADPCT:GAUGE:120:0:300' \ 
'DS:BCHARGE:GAUGE:120:0:300' \
'DS:TIMELEFT:GAUGE:120:0:128' \
'DS:BATTV:GAUGE:120:0:50' \
'DS:NUMXFERS:DERIVE:120:0:U' \ 
'DS:TONBATT:GAUGE:120:0:U' \
'RRA:LAST:0.5:1:5256000'^`
```

The problem I have is that, with my modifications or not, the output of graph-apcupsd.sh is the same:
```
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
ERROR: Unused Arguments "" in command : AREA:ds0#648B2590:Line voltage     :
convert-im6.q16: unable to open image `/usr/lib/cgi-bin/apcupsd/1hs.png': No such file or directory @ error/blob.c/OpenBlob/2701.
convert-im6.q16: unable to open file `/usr/lib/cgi-bin/apcupsd/1hs.png' @ error/png.c/ReadPNGImage/3922.
convert-im6.q16: no images defined `/usr/lib/cgi-bin/apcupsd/favicon.png' @ error/convert.c/ConvertImageCommand/3258.
```

Could you think about some further tests to debug this? Attached .zip with the 2 modified scripts..

Thanks a lot,

[graph-apcupsd.zip](https://github.com/fld/graph-apcupsd/files/754429/graph-apcupsd.zip)

**Comment by fld** [comment]
Hi, and sorry for taking so long to respond.

I've only ever tested this script on Debian 8 with RRDtool 1.4.8.

> The problem I have is that, with my modifications or not, the output of graph-apcupsd.sh is the same

Your modifications should be Ok, but since the script doesn't even work unmodified... I'm guessing that somewhere along the way of RRDtool v1.4.8->v1.6.0, there has been a subtle non-backward-compatible change in the way 'rrdtool graph' parses arguments.

I wonder if it might have something to do with this?:
https://github.com/oetiker/rrdtool-1.x/issues/655

**Comment by Lazza** [comment]
I fixed the issue on Ubuntu 16.04. The final `:` is problematic because it's a separator.

```
"AREA:ds0${PAN4}90:Line voltage     :" \
```

Remove it on the _Line voltage_ and _Output voltage_ lines:

```
"AREA:ds0${PAN4}90:Line voltage     " \
[...]
"LINE1:ds2$PAN5:Output voltage   " \
```

Now it works flawlessly.

**Comment by fld** [comment]
> I'm on Debian SID, and, 1st I had to add #!/bin/bash at the beginning of cron-apcupsd.sh to have it work.

Fixed: dadcd81

> Remove it on the Line voltage and Output voltage lines:
> 
> "AREA:ds0${PAN4}90:Line voltage     " \
> [...]
> "LINE1:ds2$PAN5:Output voltage   " \
> 
> Now it works flawlessly.

Yup, works for me as well. 
Fixed: e96bda2
